jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,120 @@
+| Runs                    | The count of executions in a selected time window. |
+| Total Duration          | The sum of execution times for all workflows in a project. |
+| P95 Duration            | The 95th percentile of execution times for an entity in a selected time window (i.e. 95% of runs completed in the same or less amount of this time). _The 95th percentile is a standard measure used to interpret performance data. It provides a measure of max value when outlier or transient values are excluded._ |
+| P50 (median) Duration   | The median execution time. _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_. |
+| Total Credits           | The sum of credits consumed during execution. |
+| Success Rate            | The percentage of runs that completed successfully, calculated by 100 x (Successful Runs / All Runs) |
+{: class="table table-striped"}
+
+## Organization-level metrics
+
+Organization-level metrics allow you to analyze your organization’s performance.
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Workflow Runs | The count of all workflows executions for an organization for the selected projects in the selected time frame. |
+| Total Workflow Duration | The total execution time of all workflows runs for an organization within the selected projects and time frame. |
+| Total Credits | The total credits consumed across all selected projects in an organization. |
+| Overall Success Rate | The percentage of runs that completed successfully across all runs in the selected projects and workflows. (calculated by `100 x (All Successful Runs/ All Runs)`) |
+{: class="table table-striped"}
+
+## Workflow Metrics
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Time to Recovery | Total time spent in a failed state before a successful execution. |
+| Mean Time to Recovery (MTTR)| The average time it takes to get back to a successful execution from a failed one (Calculated by `total recovery time / # of failures`). If there are no failed builds (and thus no recoveries) in the current time window, the MTTR value will be empty. Empty MTTR values will be treated as 0 when calculating Trends. |
+| Throughput | The average (mean) number of builds of any status per day. Calculated by `Sum of builds/ # of days.` |
+{: class="table table-striped"}
+
+
+## Trends
+
+Raw metric values are often difficult to interpret on their own. Trends provide
+additional context to Insights metrics by presenting a relative benchmark
+against previous performance. For instance, on the last 7-day view, trends will
+display the change in value or delta compared to the prior 7-day window.
+
+Please note that trends are available only for 24-hour, 7-day, and 30-day time
+windows. For percentile metrics like `duration`, approximation methods are used
+to find the prior window benchmarks.
```

Comment:
Can we add the following section below the formula? Additionally, it looks like the sentence "For percentile metrics like duration, approximation methods are used to find the prior window benchmarks." can be removed following the sentence that starts with "Please note that trends..."


"**Approximate Trends**
For percentile metrics like duration, approximation methods are used to find the prior window benchmarks. 
"